test/e2e/e2e_coverage_and_cleanup_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var workerNodeA = "kind-worker"
+var workerNodeB = "kind-worker2"
+var cmCovDir = ".coverage/clickhouse-monitor-coverage"
+var tmCovDir = ".coverage/theia-manager-coverage"
+
+func copyCovFolder(nodeName, covDir, covPrefix string) error {
+	covDirAbs, err := filepath.Abs("../../" + covDir)
+	if err != nil {
+		return fmt.Errorf("copyCovFolder: error creating absolute file path: %v", err)
+	}
+	pathOnNode := nodeName + ":" + "/var/log/" + covPrefix + "-coverage/."
+	cmd := exec.Command("docker", "cp", pathOnNode, covDirAbs)
+	var errb bytes.Buffer
+	cmd.Stderr = &errb
+	if err := cmd.Run(); err != nil {
+		errStr := errb.String()
+		if !strings.Contains(errb.String(), "Could not find the file") {
+			return fmt.Errorf("error while running docker cp command[%v] from node: %s: %s", cmd, nodeName, errStr)
+		}
+	}
+	return nil
+}
+
+func copyCovFilesBothNodes(covPrefix string) error {
+	log.Infof("Copying coverage files from worker nodes %s and %s", workerNodeA, workerNodeB)
+	if err := copyCovFolder(workerNodeA, cmCovDir, covPrefix); err != nil {
+		return err
+	}
+	if err := copyCovFolder(workerNodeA, tmCovDir, covPrefix); err != nil {
+		return err
+	}
+	if err := copyCovFolder(workerNodeB, cmCovDir, covPrefix); err != nil {
+		return err
+	}
+	if err := copyCovFolder(workerNodeB, tmCovDir, covPrefix); err != nil {
+		return err
+	}
+	return nil
+}
+
+func clearCovFolder(nodeName, covPrefix string) error {
+	nestedCmd := "`rm -rf /var/log/" + covPrefix + "-coverage/*`"
+	cmd := exec.Command("docker", "exec", nodeName, "sh", "-c", nestedCmd)
+	var errb bytes.Buffer
+	cmd.Stderr = &errb
+	if err := cmd.Run(); err != nil {
+		errStr := errb.String()
+		if !strings.Contains(errb.String(), "not found") {
+			return fmt.Errorf("error while running docker exec command[%v] from node: %s: %s", cmd, nodeName, errStr)
+		}
+	}
+	return nil
+}
```

Comment:
Are we able to combine reuse the code in `copyCovFolder`. I feel they are quite similar?